No compute example (#207)
terraform-ibm-modules/terraform-ibm-landing-zone: examples/no-compute-example/README.md

# No compute example

> This example shows how the landing-zone module can be used to create a networking infrastructure layer, without any compute resource (no VSI, nor OpenShift cluster).

Reviewer: ```suggestion
This example shows how you can use the landing zone module to create a networking infrastructure layer without any compute resources (no VSI, nor OpenShift cluster).
```

> This example deploys all of the network components and associated compliance services defined in the [mixed](https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone/tree/main/patterns/mixed) pattern - but does not deploy ANY compute resource (no VSI, nor any OpenShift cluster).

Reviewer: ```suggestion
The example deploys all the network components and associated compliance services that are defined in the [mixed](https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone/tree/main/patterns/mixed) pattern, but does not deploy any compute resource (no VSI or any OpenShift cluster).
```

> This network layer can be used as a base upon which compute resources are deployed, possibly through separate terraform modules. As such, this examples provides a base to a modular solution.

Reviewer: ```suggestion
This example provides a base to a modular solution because you can use this network layer as a base on which the compute resources are deployed, possibly through separate Terraform modules.
```

Example usage:

> ```

Reviewer: ```suggestion
```bash
```
